lib/controller/component/type/answers/answers.controller.js

```diff
@@ -325,13 +346,19 @@ answersComponent.preUpdateContents = (componentInstance, userData, pageInstance)
           delete answerBundle.actions
         }
 
-        if (onlyShowCompletedAnswers) {
-          if (!unansweredQuestion) {
-            answerBucket.push(answerBundle)
-          }
-        } else {
-          answerBucket.push(answerBundle)
-        }
+        // if (onlyShowCompletedAnswers) {
+        //   if (visitedPages.filter(visited => {
+        //     const discreteInstance = getDiscreteInstance(_id) || {}
+        //     return visited._id === discreteInstance._id
+        //   }).length) {
+        //     answerBucket.push(answerBundle)
+        //   }
+        //   // if (!unansweredQuestion) {
+        //   //   answerBucket.push(answerBundle)
+        //   // }
+        // } else {
```

Comment:
I'm guessing these comments don't need to be here

Reply:
Yep - you're not wtong